pkg/cmd/cluster/cluster_setup.go

```diff
@@ -18,11 +16,10 @@ func newCmdClusterSetup(ctx api.Context) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			clusterURL := args[0]
 
-			// This supports passing a cluster URL without scheme, it then defaults to HTTPS.
-			if !strings.HasPrefix(clusterURL, "https://") && !strings.HasPrefix(clusterURL, "http://") {
-				clusterURL = "https://" + clusterURL
-			}
-			return ctx.Setup(setupFlags, clusterURL)
+			// We want to attach the cluster once it is setup.
+			attach := true
+			_, err := ctx.Setup(setupFlags, clusterURL, attach)
+			return err
```

Comment:
I find it clearer without the `attach` argument, it removes the need to pass a variable for readability and avoids the blank-assignment, while not being much more verbose : 
``` go
config, err := ctx.Setup(setupFlags, clusterURL)
if err != nil {
    return err
}
return ctx.ConfigManager().Attach(config)
```